R/spark_compile.R

```diff
@@ -126,7 +126,7 @@ spark_compile <- function(name,
 #'
 #' @keywords internal
 #' @export
-spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
+compile_package_jars <- function(package = rprojroot::find_package_root_file(),
                                        spark_versions = NULL,
                                        scalac = NULL,
```

Comment:
We try to find the program using `path_program` when this is `NULL` below, although perhaps we should just make the default `path_program("scalac")` instead of `NULL`.